packages/default-theme/components/SwBottomNavigation.vue

```diff
@@ -54,6 +59,7 @@ import {
   useUserLoginModal
 } from '@shopware-pwa/composables'
 import { PAGE_ACCOUNT } from '../helpers/pages'
+import SwCurrency from './SwCurrency'
```

Comment:
we need to import them by aliases, so `@shopware-pwa/default-theme.....`

Reply:
:+1: